src/JuMPModels/space_Shuttle.jl

```diff
@@ -0,0 +1,151 @@
+import Interpolations
```

Comment:
We should remove `Interpolations` from the deps. The space shuttle instance uses a simple linear interpolations we can code manually.